test/extended/builds/forcepull.go

```diff
@@ -79,17 +89,12 @@ we built.  The credentials also are recycled between those points.
 Dumping of the ImageStreams and Secrets JSON output at the various points proved this out.
 */
 
-var _ = g.Describe("[LocalNode][builds] forcePull should affect pulling builder images", func() {
+var _ = g.Describe("[builds] forcePull should affect pulling builder images", func() {
 	defer g.GinkgoRecover()
 	var oc = exutil.NewCLI("forcepull", exutil.KubeConfigPath())
 
 	g.BeforeEach(func() {
 
-		g.By("refresh corruptor, prep forcepull builder")
-		exutil.PullImage(corruptor, dockerClient.AuthConfiguration{})
-
-		exutil.DumpImage(corruptor)
-
 		// create the image streams and build configs for a test case specific builders
 		setupPath := exutil.FixturePath("testdata", "forcepull-setup.json")
 		err := exutil.CreateResource(setupPath, oc)
```

Comment:
is all this setup logic still needed, now that we aren't corrupting images?  (and thus don't need to worry about conflicting with other tests that are running on the node)?